src/Batcher/Database.ts

```diff
@@ -0,0 +1,28 @@
+import { inject, injectable } from 'inversify'
+import { Collection, Db } from 'mongodb'
+
+@injectable()
+export class Database {
+  private readonly db: Db
+  private readonly collection: Collection
+
+  constructor(@inject('DB') db: Db) {
+    this.db = db
+    this.collection = this.db.collection('batcher')
+  }
+
+  addItem = async ({ item }: { item: string }) => await this.collection.insertOne({ item, time: null })
```

Comment:
@warrenv @wzalazar we could all the functions in this file with integration tests by mocking mongo, is that something you guys would like from now on?